triage/storage.py

```diff
@@ -128,6 +128,21 @@ def labels(self):
     def uuid(self):
         return self.metadata['metta-uuid']
 
+    def columns(self, include_label=False):
+        columns = self.matrix.columns.tolist()
+        if include_label:
+            return columns
+        else:
+            return [
+                col for col in columns
+                if col != self.metadata['label_name']
+            ]
+
+    def matrix_with_sorted_columns(self, columns):
+        if self.columns() != columns:
+            logging.warning('Column orders not the same, re-ordering')
+        return self.matrix[columns]
```

Comment:
I think I see what you mean: We only want to proceed if the `set` versions of both are the same. The log message here would be shown if that is true *but* the ordering is not the same.